src/utils/vc/crypto/common/DockCryptoSignatureProof.js

```diff
@@ -102,7 +103,6 @@ export default withExtendedStaticProperties(
         type,
         credentialSchema,
         credentialStatus,
-        issuer: _issuer,
```

Comment:
why was this removed? could it make older credentials incompatible or something?

Reply:
Good point. It can cause issue with verifying older presentations but not credentials. Fixed.